### PR TITLE
Basic scaffold implementation

### PR DESCRIPTION
**Pull request by jrobind**
I've started a basic scaffold to get things under way. The webpack config will most definitely need some modification.

I've created a `__tests__` directory for now, but maybe we should discuss what test suite we would like to use (if at all)?

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            }}, 'sass-loader']},
+            {test: /\.(png|jpg|svg)$/, loader: 'url-loader?linit=8192'}
+        ]
+    },
+    devServer: {
+        historyApiFallback: true  
+    },
+    plugins: [
+        new htmlWebpackPlugin({
+            template: 'app/index.html',
```
Comment (Mattedigital) [inline webpack.config.js]: Could add html page title in here: https://github.com/jantimon/html-webpack-plugin/issues/612

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        extensions: ['.js', '.jsx']
+    },
+    module: {
+        rules: [
+            {test: /\.(js|jsx)$/, use: 'babel-loader', exclude: /node_modules/},
+            {test: /\.scss$/, use: ['style-loader', {loader: 'css-loader', options: {
+                modules: true,
+                localIdentName: '[hash:base64:5]__[local]'
+            }}, 'sass-loader']},
+            {test: /\.(png|jpg|svg)$/, loader: 'url-loader?linit=8192'}
```
Comment (Mattedigital) [inline webpack.config.js]: Need to be sure not to use .jpeg or .gif. Or add in?

File: package.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "spotify_workplace_app",
+  "version": "1.0.0",
+  "description": "A custom music app utlising the Spotify api",
```
Comment (Mattedigital) [inline package.json]: *utilising

File: webpack.config.js (hunk 2 above)
Comment (jrobind) [reply webpack.config.js]: Yeah I agree, definitely worth adding these in.

File: webpack.config.js (hunk 1 above)
Comment (jrobind) [reply webpack.config.js]: Didn't know you could do this as part of the config - awesome.

**Comment by jrobind** [comment]
I've made the relevant changes!

 I'm hoping to have some time free over the weekend to send over another pull request, hopefully with the config nearer completion.

**Pull request merged**